<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamescreen</class>
 <widget class="QMainWindow" name="gamescreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1831</width>
    <height>1016</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>585</height>
    </size>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>1271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: yellow;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>1241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: lightblue;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>20</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: yellow;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: lightblue;
;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: yellow;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>1311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: gold;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>20</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: blue;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>16</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: gold;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>16</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: lightblue;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_5">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>0</y>
      <width>20</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: yellow;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_6">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>70</y>
      <width>20</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: blue;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_7">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>20</y>
      <width>20</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: lightblue;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_8">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>40</y>
      <width>20</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: gold;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1241</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>380</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label10">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>390</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label38">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>280</y>
       <width>60</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label40">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>360</y>
       <width>60</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>390</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label28">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label39">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>320</y>
       <width>60</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label37">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>230</y>
       <width>60</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label25">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label30">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label23">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label27">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label24">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label26">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label29">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>390</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label16">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label36">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>205</y>
       <width>60</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label7">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>390</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label17">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label34">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>120</y>
       <width>60</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label35">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>167</y>
       <width>60</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label6">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>390</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label21">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label22">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>390</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label33">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>78</y>
       <width>60</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label19">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>380</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label32">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>52</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label31">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>12</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>390</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>1171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider{
	color: lightblue;
	background: dark blue;
	
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>1281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider{
	color: lightblue;
	background: blue;
	
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>841</width>
       <height>261</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Please select the number of players and then hit OK!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>287</width>
        <height>20</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>  2 Players</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>  3 Players</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string>  4 Players</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>label_2</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>groupBox</zorder>
     <zorder>horizontalSlider</zorder>
     <zorder>horizontalSlider</zorder>
     <zorder>horizontalSlider_2</zorder>
     <zorder>horizontalSlider_2</zorder>
     <zorder>horizontalSlider_3</zorder>
    </widget>
    <zorder>label4</zorder>
    <zorder>label10</zorder>
    <zorder>label38</zorder>
    <zorder>label40</zorder>
    <zorder>label15</zorder>
    <zorder>label11</zorder>
    <zorder>label28</zorder>
    <zorder>label39</zorder>
    <zorder>label37</zorder>
    <zorder>label25</zorder>
    <zorder>label30</zorder>
    <zorder>label23</zorder>
    <zorder>label27</zorder>
    <zorder>label24</zorder>
    <zorder>label26</zorder>
    <zorder>label29</zorder>
    <zorder>label14</zorder>
    <zorder>label2</zorder>
    <zorder>label16</zorder>
    <zorder>label36</zorder>
    <zorder>label17</zorder>
    <zorder>label34</zorder>
    <zorder>label35</zorder>
    <zorder>label21</zorder>
    <zorder>label12</zorder>
    <zorder>label18</zorder>
    <zorder>label22</zorder>
    <zorder>label3</zorder>
    <zorder>label33</zorder>
    <zorder>label19</zorder>
    <zorder>label32</zorder>
    <zorder>label20</zorder>
    <zorder>label13</zorder>
    <zorder>label31</zorder>
    <zorder>label1</zorder>
    <zorder>label7</zorder>
    <zorder>label5</zorder>
    <zorder>label8</zorder>
    <zorder>label9</zorder>
    <zorder>label6</zorder>
    <zorder>horizontalSlider_4</zorder>
    <zorder>horizontalSlider_5</zorder>
    <zorder>label_2</zorder>
    <zorder>widget</zorder>
    <zorder>widget</zorder>
    <zorder>widget</zorder>
    <zorder>horizontalSlider_3</zorder>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>1211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color: lightblue;
	background: gold
;
	
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>horizontalSlider_6</zorder>
   <zorder>horizontalSlider_7</zorder>
   <zorder>verticalSlider_2</zorder>
   <zorder>verticalSlider_3</zorder>
   <zorder>verticalSlider_4</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>horizontalSlider_2</zorder>
   <zorder>horizontalSlider_8</zorder>
   <zorder>verticalSlider_6</zorder>
   <zorder>verticalSlider_5</zorder>
   <zorder>verticalSlider_7</zorder>
   <zorder>verticalSlider_8</zorder>
   <zorder>horizontalSlider_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
